app/helpers/authorization_requests_helpers.rb

```diff
@@ -1,12 +1,16 @@
 module AuthorizationRequestsHelpers
+  def hidden_attributes
+    params.slice(:attributes).permit!
+  end
+
```

Comment:
peu fan de ce nom, et cf mon com dans la vue en vrai je pense que tu peux harden ici avec un truc du genre:

```suggestion
  def new_authorization_request_default_params(authorization_request)
    return {} if authorization_request.persisted?
    
    params[:attributes]
  end

```

Reply:
si je permit! pas, rails lâche une erreur. Mais je vois l'idée